frontend/express/public/javascripts/countly/countly.common.js

```diff
@@ -4121,7 +4123,14 @@
                 return "0";
             }
             else {
-                return resultStrings.join(" ");
+                let res = resultStrings[0];
+                if (trimTo > 3 || trimTo < 1) {
+                    trimTo = 3;
+                }
+                for (let j = 1; j < trimTo && j < resultStrings.length; j++) {
+                    res += ' ' + resultStrings[j];
+                }
```

Comment:
I think this loop can be replaced by array.slice which will be more error prone and more understandable, sorry :)